### ISSUE TITLE
Accessibility issue - "Beep on caps/num lock" does not work

### ISSUE THREAD

**Issue by tonywhelan** [linuxmint/cinnamon-control-center#95]
Mint 17 Qiana Cinnamon 64 bit.  
In System Settings, Accessibility, "Seeing" tab, when you enable the "Beep on Caps and Num Lock" the computer does NOT beep (as it should) when those Lock keys are pressed.

This is also the case in Ubuntu 14.04 and several prior versions.

I have verified this feature does not work on 4 different machines with LM Cinnamon 17 installed, incl 1 laptop.

It appears the current code inherited from Ubuntu is intended to sound the "beep" via the PC speaker. However:
1. Some systems (eg lapops) don't have a PC speaker:
2. Mint by default blacklists the PC speaker as its often too loud and ugly
3. Having the same sound for Caps Lock ON and Caps Lock OFF is pointless; we need a separate sound for On and OFF key states.

Its a trivial matter for someone with a bit of knowledge to set up a key binding and a script that will play a beep when a Lock key is pressed ON, and play a different (lower-toned) beep when pressed OFF. But it shouldn't be necessary - the facility should work out of the box. 

Suggest the Accessibility module code be changed as follows:
1.  Move the "Beep" option from the "Seeing" tab to the more logical "Hearing" tab.
2.  Add suitable "on" and "off" sounds to /usr/share/mint-artwork-cinnamon/sounds;  ON should be a higher pitched tone than OFF.
3.  Ensure that pressing Caps Lock or Num Lock ON or OFF plays the appropriate tone.

Fixing the above bug would be appreciated not just by people with disability restrictions, but also by anyone using keyboards or laptops that have no Caps Lock or Num Lock light, or have a light that's too small to be useful.

**Comment by tonywhelan** [comment]
Closing as have posted under /Cinnamon
